translations/kubectl/ko_KR/LC_MESSAGES/k8s.po

```diff
@@ -0,0 +1,89 @@
+msgid "Apply a configuration to a resource by filename or stdin"
+msgstr "구성을 파일 이름 또는 stdin에 의한 자원에 적용합니다"
+
+# https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/config/delete_cluster.go#L38
+msgid "Delete the specified cluster from the kubeconfig"
+msgstr "kubeconfig에서 지정된 클러스터를 삭제합니다"
+
+# https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/config/delete_context.go#L38
+msgid "Delete the specified context from the kubeconfig"
+msgstr "kubeconfig에서 지정된 컨텍스트를 삭제합니다"
+
+# https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/config/get_contexts.go#L62
+msgid "Describe one or many contexts"
+msgstr "하나 또는 여러 컨텍스트를 설명합니다"
+
+# https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/config/get_clusters.go#L40
+msgid "Display clusters defined in the kubeconfig"
+msgstr "kubeconfig에 정의된 클러스터를 표시합니다"
+
+# https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/config/view.go#L64
+msgid "Display merged kubeconfig settings or a specified kubeconfig file"
+msgstr "병합된 kubeconfig 설정 또는 지정된 kubeconfig 파일을 표시합니다"
+
+# https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/config/current_context.go#L48
+msgid "Displays the current-context"
+msgstr "현재-컨텍스트를 표시합니다"
+
+# https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/config/config.go#L39
+msgid "Modify kubeconfig files"
+msgstr "kubeconfig 파일을 수정합니다"
+
+# https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/config/create_cluster.go#L67
+msgid "Sets a cluster entry in kubeconfig"
+msgstr "kubeconfig에서 클러스터 항목을 설정합니다"
+
+# https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/config/create_context.go#L57
+msgid "Sets a context entry in kubeconfig"
+msgstr "kubeconfig에서 컨텍스트 항목을 설정합니다"
+
+# https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/config/create_authinfo.go#L103
+msgid "Sets a user entry in kubeconfig"
+msgstr "kubeconfig에서 사용자 항목을 설정합니다"
+
+# https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/config/set.go#L59
+msgid "Sets an individual value in a kubeconfig file"
+msgstr "kubeconfig 파일에서 단일값을 설정합니다"
+
+# https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/config/use_context.go#L48
+msgid "Sets the current-context in a kubeconfig file"
+msgstr "kubeconfig 파일에서 현재-컨텍스트를 설정합니다"
+
+# https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/config/unset.go#L47
+msgid "Unsets an individual value in a kubeconfig file"
+msgstr "kubeconfig 파일에서 단일값 설정을 해제합니다"
+
+msgid "Update the annotations on a resource"
+msgstr "자원에 대한 주석을 업데이트합니다"
+
+msgid ""
+"watch is only supported on individual resources and resource collections - "
+"%d resources were found"
+msgid_plural ""
+"watch is only supported on individual resources and resource collections - "
+"%d resources were found"
+msgstr[0] ""
+"watch는 단일 리소스와 리소스 모음만을 지원합니다 - %d 개 자원을 발견하였습"
```

Comment:
Typo. Plz join these 2 lines.

Reply:
@yue9944882 That's because it is recommended to wrap lines at column 80. Would it be okay?

[1] http://pology.nedohodnik.net/doc/user/en_US/ch-poformat.html

Reply:
It's okay. I looks shorter than L85 tho. Maybe due to github displaying problem.